js_modules/dagit/packages/core/src/assets/AutoMaterializePolicyPage/Conditions.tsx

```diff
@@ -83,7 +86,10 @@ export const ConditionsWithPartitions = ({
           />
         </Box>
       </CollapsibleSection>
-      <CollapsibleSection header="Skip conditions met">
+      <CollapsibleSection
+        header="Skip conditions met"
+        details="These conditions block materializations until they are resolved."
```

Comment:
comprehensible comparison of skip and discard?

Reply:
Yeah the difference is somewhat hard to explain -- I think the core difference is somewhat along the lines of:

Skip: This asset is expected to be materialized at a later point for the same materialization reason, no manual action necessary

Discard: This asset will not be materialized at a later point for the same materialization reason, some manual work will need to be done to have this asset / partition materialized

Reply:
Maybe:

Skip: "Skipped partitions will not be materialized, will be materialized for the same materialization reason once the condition is resolved."

Discard: "Discarded partitions will not be materialized, and will not be re-evaluated for the same materialization reason once the condition is resolved"

Reply:
Maybe mention a backfill?

`Skipped partitions will materialize in a future evaluation once the skip condition is resolved.`

`Discarded partitions will not be materialized unless new future materialization conditions occur. You may want to backfill these partitions.`

Reply:
Could also take the approach of explaining why we have discard

`To avoid large numbers of accidental runs, partitions beyond the limit are discarded. They will not materialize for the same materialization condition`.

Reply:
I'm ok with:

`Skipped partitions will be materialized in a future evaluation, once the skip condition is resolved.`

`Discarded partitions will not be materialized unless new materialization conditions occur in the future. These may need to be backfilled manually to respond to the materialize conditions.`

or something like that for now

Reply:
updated to close to this